package/base-files/files/lib/upgrade/mmc.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/sh
```

Comment:
That's a library file, remove the shebang and the execute bit (if there is one).

Reply:
Removed, thanks!